Ref/Top/RefTopology.hpp

```diff
@@ -78,7 +78,7 @@ void teardownTopology(const TopologyState& state);
  *
  * \param milliseconds: milliseconds to delay for each cycle. Default: 1000 or 1Hz.
  */
-void startSimulatedCycle(U32 milliseconds = 1000);
+void startSimulatedCycle(Fw::Time interval);
```

Comment:
The default argument went away, but it's still in the comment. Should we put the default argument back?

Reply:
Or we could eliminate the default argument (which seems somewhat arbitrary) and update the comment.

Reply:
We will fix the comment.